webhookv3/webhook_event.go

```diff
@@ -13,6 +13,7 @@ type OutboundEventData struct {
 	Object map[string]interface{}
 	// The raw json data for use in structured unmarshalling.
 	RawData json.RawMessage
+	Type    string
```

Comment:
I'm not currently making this of type `EventDataType` as that would break support unstructured event handling.